### PR TITLE
ProposedSolution to Issue #2048

### PR DESCRIPTION
**Pull request by AZenker**
To support CIFS:
Updated these things in class NistMirrorTask:
(1) Replaced Files.Move with FIles.copy and Files.delete to avoid the exception
(2) Avoided "LastModified" attribute instead a timestamp file is created to save this modification time and reuse the time from this file in the rest of the workflow

### CODE REVIEW

**Comment by nscuro** [comment]
Thanks for the PR @AZenker! 

We'll need you to sign-off your commits before we can merge. If you click on *Details* of the DCO check it'll show you what you have to do.

**Comment by AZenker** [comment]
> File(outputDir, filename).getAbsoluteFile();

Ok, I have updated the commit message! :-)

**Review by nscuro — APPROVED** [review]

**Comment by nscuro** [comment]
FTR, the test failure is not related to this change.

```
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 5.248 s <<< FAILURE! - in org.dependencytrack.tasks.BomUploadProcessingTaskTest
Error:  informTest(org.dependencytrack.tasks.BomUploadProcessingTaskTest)  Time elapsed: 4.471 s  <<< FAILURE!
java.lang.AssertionError: 

Expected size: 2 but was: 1 in:
[org.dependencytrack.model.Vulnerability@374bbe80]
	at org.dependencytrack.tasks.BomUploadProcessingTaskTest.informTest(BomUploadProcessingTaskTest.java:148)
```

**Pull request merged**
